.github/workflows/integration-test.yml

```diff
@@ -120,25 +88,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Free Disk Space
-        uses: jlumbroso/free-disk-space@v1.3.1
-        with:
-          # Runs quick, run rm in background
-          tool-cache: true
-
-          # These run slower as use apt to uninstall packages
-          # Turned on by default, so we disable them
-          android: false
-          dotnet: false
-          haskell: false
-          large-packages: false
-          docker-images: false
-          swap-storage: false
-
       - name: Run UI Tests
         uses: dagger/dagger-for-github@v6
         with:
           verb: call
           version: ${{ env.DAGGER_VERSION }}
           args: test --source .:default ui
-          cloud-token: ${{ secrets.DAGGER_CLOUD_TOKEN }}
```

Comment:
i decided to remove Dagger Cloud because im not sure we're benefiting too much from it and I also don't think this free-disk-space action works incredibly well. The disk space usage is caused by Dagger Cloud downloading the entire cache history (I think) on run, which both slows down CI and blows up our disk.